.semaphore/pre-release.yml

```diff
@@ -23,7 +23,7 @@ blocks:
         - name: "Publish to NPM"
```

Comment:
Would it be more clear to call this file `.semaphore/alpha-release.yml`, and change line 23 to that as well, and update the scheduler on semaphore? Maybe it's just me, but i think this name insinuates that this file gets run immediately before `.semaphore/release.yml` -- which it does not.

Reply:
I see, how about `next-release.yml`? because there won't be any `alpha` versions anymore, will be `next` versions instead

Reply:
Fine by me, if it makes sense to you and @nahumzs  (also on line 29 in this file maybe update that commit message as well)

Reply:
@nahumzs allison and i chatted and agreed to leave it as `pre-release` since it's the industry standard term.